arch/riscv/riscv_features.c

```diff
@@ -45,6 +52,7 @@ void Z_INTERNAL riscv_check_features_runtime(struct riscv_cpu_features *features
     unsigned long hw_cap = 0;
 #endif
     features->has_rvv = hw_cap & ISA_V_HWCAP;
+    features->has_crc32 = hw_cap & ISA_ZBC_HWCAP;
```

Comment:
Should this be has_zbc which is the name of the instruction set rather than crc32 which is the name of the algorithm?